tests/Client/GroupsClientTest.php

```diff
@@ -34,69 +34,104 @@ public function testCanBeConstructed(): void
         $this->assertInstanceOf(GroupsClient::class, $client);
     }
 
-    public function testBuildApiUrl(): void
+    /**
+     * @dataProvider buildApiUrlProvider
```

Comment:
Can we use attributes? Because of 
```
1) Metadata found in doc-comment for method Client\GroupsClientTest::testBuildApiUrl(). Metadata in doc-comments is deprecated and will no longer be supported in PHPUnit 12. Update your test code to use attributes instead.
```

Reply:
sure

Reply:
Please merge 
* https://github.com/slunak/pushover-php/pull/84

first, thanks